app/build.gradle

```diff
@@ -2,22 +2,31 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'realm-android'
 apply plugin: 'deploygate'
+apply plugin: 'com.github.ben-manes.versions'
+apply plugin: 'com.github.triplet.play'
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.1"
+    buildToolsVersion '26.0.2'
+    playAccountConfigs {
+        defaultAccountConfig {
+            jsonFile = file('app-publisher.json')
+        }
+    }
     defaultConfig {
         applicationId "com.mataku.scrobscrob"
 
         minSdkVersion 24
         targetSdkVersion 26
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/build.gradle#L20"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>